price-feeder/README.md

```diff
@@ -48,6 +48,20 @@ $ price-feeder /path/to/price_feeder_config.toml
 ```
 
 ## Configuration
+There are multiple new options for validators to setup in the price feeder config, the options are telemetry, deviation, and custom endpoints. 
+
+### `Telemetry`
+• A set of options for the /metrics endpoint when running the price feeder. 
```

Comment:
Let's try to be a bit more descriptive, use the same case they would in the config file (e.g., `deviation_thresholds` or `telemetry`, and add the same spacing we're using on other parts of the config list:
```suggestion
### `telemetry`

A set of options for the application's telemetry, which is disabled by default. An in-memory sink is the default, but Prometheus is also supported.
```

Maybe it'd be helpful to also link to the config here?